src/main/kotlin/menu/Menu.kt

```diff
@@ -0,0 +1,82 @@
+package Menu
+import Items.Item
+import TypeOfInterface
+import Utils
+
+abstract class Menu<T,I : Item<T>>(titleOfMenu:String, typeOfInterface: TypeOfInterface, itemList:MutableList<I>) {
+
+    private val titleOfMenu:String = titleOfMenu //установим заголовок меню
+
+    private var itemList:MutableList<I> = itemList//Общий список хранения объектов Архивы Заметки и тд
+    var listOfMenu:MutableMap<Int,String> = mutableMapOf<Int, String>() //Список для вывода ИНДЕКС и Наименование команды или объекта
```

Comment:
```suggestion
    val listOfMenu: MutableMap<Int,String> = mutableMapOf() //Список для вывода ИНДЕКС и Наименование команды или объекта
```
Явное задание типов в mutableMapOf лишнее